effortless-chef-io/effortless-audit.md

```diff
@@ -118,36 +132,45 @@ In InSpec a common pattern is to write a wrapper profile that depends on another
    cd $pkg_prefix
    hab pkg exec $pkg_origin/$pkg_name inspec exec $pkg_prefix/*.tar.gz
    ```
-1. Run Test Kitchen to ensure your profile executes
-   ```
+
+1. Run Test Kitchen to ensure your profile executes:
+
+   ```shell
    kitchen converge base-centos
    ```
-   > Note: This will spin up a CentOS 7 VM locally and run you profile using the latest Chef InSpec. If you get failures that's okay most vanilla VM's are not fully hardened to your security policies. If you want to fix the failures take a look at using [Chef Infra and the Effortless Config Pattern](effortless-config.md).
-1. When you are ready destroy the VM by running `kitchen destroy`
-1. You can now upload your profile pkg to builder by running the following
-   ```
+
+   > Note: This will spin up a CentOS 7 VM locally and run you profile using the latest Chef Inspec. If you get failures that's okay most vanilla VMs are not fully hardened to your security policies. If you want to fix the failures take a look at using [Chef Infra and the Effortless Config Pattern](effortless-config.md).
+1. When you are ready, destroy the VM by running `kitchen destroy`
+1. You can now upload your profile pkg to Chef Habitat Builder by running:
+
+   ```shell
    source results/lastbuild.env
    hab pkg upload results/$pkg_artifact
    ```
-1. To run your profile on a system you just need to install habitat as a service and run `hab svc load <your_origin>/<your_profile_name>`
+
+1. To run your profile on a system you just need to install habitat as a service and run `hab svc load <my_origin>/<_profile_name>`
 
 ## Features
 
 ### Waivers
 
-With the release of scaffolding-chef-inspec version 0.16.0 (Linux) and version 0.18.0 (Windows) we have added the Chef InSpec waivers feature. This feature allows you to specify a control id's in your habtitat config that you would like to skip/waive. 
+We added the Chef Inspec waivers feature to Chef Compliance with scaffolding-chef-inspec version 0.16.0 (Linux) and version 0.18.0 (Windows) release. The waivers feature allows you to specify control IDs in your Chef Habitat configuration that you would like to skip/waive.
+
+1. Build an Effortless Audit profile and load it on your systems
+1. Create a `waiver.toml` file like the below example
+
+    ```toml
+    [waivers]
+    [waivers.control_id]
+    run = false
+    expiration_date: 2021-11-31
```

Comment:
@kagarmoe These updates look great, was there a reason for the highlighting on this line and line 167 though?

Reply:
@collinmcneese Github doesn't like it when you include one underscore and don't include a second underscore. It should be okay in the end.